cep-sigstore-predicate.md

```diff
@@ -0,0 +1,72 @@
+# CEP - Standardizing the v1 predicate for sigstore attestations
+
+<table>
+<tr><td> Title </td><td> Standardizing the v1 predicate for sigstore attestations </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Wolf Vollprecht &lt;wolf@prefix.dev&gt;</td></tr>
+<tr><td> Created </td><td> Feb 18, 2025 </td></tr>
+<tr><td> Updated </td><td> Feb 18, 2025</td></tr>
+<tr><td> Discussion </td><td>  </td></tr>
+<tr><td> Implementation </td><td>  </td></tr>
+</table>
+
+## Abstract
+
+We want to standardize attestations for the conda ecosystem.
+
+### Sigstore Attestations
+
+Sigstore attestations are cryptographic statements about software artifacts that provide:
```

Comment:
Does "SigStore attestation" here mean the SigStore bundle itself or some other attestation payload defined elsewhere, wrapped as the SigStore verification material? To reduce confusion for someone reading those specs, we should use the same [language that SigStore does](https://docs.sigstore.dev/about/bundle/).